neon_core/skills/service.py

```diff
@@ -160,13 +160,18 @@ def run(self):
         # Setup Intents and Skill Manager
         self._register_intent_services()
         self.event_scheduler = EventScheduler(self.bus)
-        self.status = ProcessStatus('skills', self.bus, self.callbacks,
-                                    namespace="neon")
         SkillApi.connect_bus(self.bus)
         LOG.info("Starting Skill Manager")
-        self.skill_manager = NeonSkillManager(self.bus, self.watchdog)
+        self.skill_manager = NeonSkillManager(
+            bus=self.bus, watchdog=self.watchdog,
+            alive_hook=self.callbacks.on_alive,
+            started_hook=self.callbacks.on_started,
+            ready_hook=self.callbacks.on_ready,
+            error_hook=self.callbacks.on_error,
+            stopping_hook=self.callbacks.on_stopping)
         self.skill_manager.setName("skill_manager")
         self.skill_manager.start()
+        self.status = self.skill_manager.status  # TODO: Deprecate backwards-compat?
```

Comment:
should this be a property instead?

Reply:
Yeah.. I'll make it a property with a deprecation warning